src/UserSpaceInstrumentation/AddressRange.h

```diff
@@ -15,6 +18,16 @@ namespace orbit_user_space_instrumentation {
 struct AddressRange {
   AddressRange() = default;
   AddressRange(uint64_t start, uint64_t end) : start(start), end(end) {}
+
+  // Make AddressRange hashable so we can use it as a key in a flat_hash_map.
+  bool operator==(const AddressRange& other) {
+    return other.start == start && other.end == end;
+  }
+  template <typename H>
+  H AbslHashValue(H h, const AddressRange& a) {
```

Comment:
According to https://abseil.io/docs/cpp/guides/hash#making-hashable-types, `AbslHashValue` is not supposed to be a class member. I guess it makes sense because otherwise both `*this` and the parameter are an `AddressRange`.

Reply:
Yep, moved it out of the class.